app/components/primer/progress_bar_component.rb

```diff
@@ -52,13 +52,14 @@ class Item < ViewComponent::Slot
       include ClassNameHelper
       attr_reader :kwargs
 
-      def initialize(percentage: 0, bg: :green, **kwargs)
+      def initialize(percentage: 0, bg: :green, style: "", **kwargs)
```

Comment:
I believe we can lean on `kwargs` here. See suggestion below 😄 

```suggestion
      def initialize(percentage: 0, bg: :green, **kwargs)
```